apps/site/pages/en/learn/modules/package-configuration.md

```diff
@@ -0,0 +1,563 @@
+
+| You as a package author write     | Consumers of your package write their code in                    | Your options                                                                                                                                                 |
+| :-------------------------------- | :--------------------------------------------------------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| CJS source code using `require()` | CJS: consumers `require()` your package                          | [CJS source and distribution](#cjs-source-and-distribution)                                                                                                  |
+| CJS source code using `require()` | ESM: consumers `import` your package                             | [CJS source and only ESM distribution](#cjs-source-and-only-esm-distribution)                                                                                |
+| CJS source code using `require()` | CJS & ESM: consumers either `require()` or `import` your package | [CJS source and both CJS & ESM distribution](#cjs-source-and-both-cjs-amp-esm-distribution)                                                                  |
+| ESM source code using `import`    | CJS: consumers `require()` your package                          | [ESM source with only CJS distribution](#esm-source-with-only-cjs-distribution)<br />[ESM source and distribution](#esm-source-and-distribution)             |
+| ESM source code using `import`    | ESM: consumers `import` your package                             | [ESM source and distribution](#esm-source-and-distribution)                                                                                                  |
+| ESM: source code uses `import`    | CJS & ESM: consumers either `require()` or `import` your package | [ESM source and both CJS & ESM distribution](#esm-source-and-both-cjs-amp-esm-distribution)<br />[ESM source and distribution](#esm-source-and-distribution) |
+
+### CJS source and distribution
+
+This the "Rum & Coke" of packages: pretty difficult to mess up. You _technially_ may not need _any_ package configuration beyond [`"name"`](https://nodejs.org/api/packages.html#name). But the less arcane, the better: Essentially just declare the package’s exports via the `"exports"` field/field-set.
+
+**Working example**: [cjs-with-cjs-distro](https://github.com/JakobJingleheimer/nodejs-module-config-examples/tree/main/packages/cjs/cjs-distro)
+
+```json displayName="Minimal package.json"
+{
+  "name": "cjs-source-and-distribution"
+  // "main": "./index.js"
+}
+```
+
+```json displayName="Advanced (verbose) package.json"
+{
+  "name": "cjs-source-and-distribution",
+  "type": "commonjs", // current default, but may change
+  "engines": { "node": ">=12.22.7" },
```

Comment:
```suggestion
```

This doesn't seem necessary?

Reply:
Nope. That's explained at the top; these are also hidden by default per your suggestion to keep things simple by default:

<img width="400" alt="Default: minimal package.json without 'engines'" src="https://github.com/user-attachments/assets/dadfed92-55a5-43a1-9327-365eb9cf3e42">

<img width="400" alt="Non-default: advanced package.json with 'engines'" src="https://github.com/user-attachments/assets/f5810b4e-2bec-4278-b42b-855fb34c0a49">

Reply:
oh right, missed the explanation about tabs, sorry.